examples/modern_data_stack_assets/README.md

```diff
@@ -1,2 +1,65 @@
+# Intro
+
 This is an example of how to use the Software-Defined Asset APIs alongside Modern Data Stack tools
 (specifically, Airbyte and dbt).
+
+# Setup
+
+## Python
+
+To install the python dependencies for this example, run:
+
+```
+$ pip install -e .
+```
+
+Once you've done this, you can run:
+
+```
+$ dagit -m modern_data_stack_assets
+```
+
+To view this example in Dagster's UI, Dagit. If you want to run this job successfully, you'll need to do a bit of extra setup, described below.
```

Comment:
"this job" is slightly confusing wording. Maybe "if you want to run the job that shows up successfully" or "if you want to materialize all the assets successfully".